docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Log appointment details to an external application in Outlook mobile add-ins
+description: Learn how to set up an Outlook mobile add-in to log appointments to an external application.
+ms.topic: article
+ms.date: 04/11/2022
+ms.localizationpriority: medium
+---
+
+# Log appointment details to an external application in Outlook mobile add-ins
+
+Saving your appointment details and related notes to a customer relationship management (CRM) or note-taking application can help you keep track of meetings you've attended.
+
+In this article, you'll learn how to set up your Outlook mobile add-in to enable users to log details and notes about their appointments to your CRM or note-taking application. Throughout this article, we'll be using a fictional service provider, "Contoso".
+
+> [!IMPORTANT]
+> This feature is only supported on Android with a Microsoft 365 subscription.
+
+## Set up your environment
+
+Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator) which creates an add-in project with the Yeoman generator for Office Add-ins.
+
+## Configure the manifest
+
+To enable users to log appointment notes with your add-in, you must configure the [MobileLogEventAppointmentAttendee extension point](/javascript/api/manifest/extensionpoint#mobilelogeventappointmentattendee) in the manifest under the parent element `MobileFormFactor`. Other form factors are not supported.
+
+1. In your code editor, open the quick start project.
+
+1. Open the **manifest.xml** file located at the root of your project.
+
+1. Select the entire `<VersionOverrides>` node (including open and close tags) and replace it with the following XML. Make sure to replace all references to **Contoso** with your company's information.
+
+```xml
+<VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides" xsi:type="VersionOverridesV1_0">
+  <VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides/1.1" xsi:type="VersionOverridesV1_1">
+    <Description resid="residDescription"></Description>
+    <Requirements>
+      <bt:Sets>
+        <bt:Set Name="Mailbox" MinVersion="1.3"/>
+      </bt:Sets>
+    </Requirements>
+    <Hosts>
+      <Host xsi:type="MailHost">
+        <MobileFormFactor>
+          <FunctionFile resid="residFunctionFile"/>
+          <ExtensionPoint xsi:type="MobileLogEventAppointmentAttendee">
+            <OfficeTab id="TabDefault">
+              <Group id="mobileLogGroup">
```

Comment:
This extension point can have just one control. Either a task pane or a ui-less function. Please refer this doc shared.
-Manbendra